client_metrics.go

```diff
@@ -18,17 +19,20 @@ type ClientMetrics struct {
 	clientStreamMsgReceived *prom.CounterVec
 	clientStreamMsgSent     *prom.CounterVec
 
-	clientHandledHistogramEnabled bool
-	clientHandledHistogramOpts    prom.HistogramOpts
-	clientHandledHistogram        *prom.HistogramVec
+	doOnceClientHandledHistogramEnable sync.Once
```

Comment:
wasn't sure what a good name is, this seems to work but if this should be named different just let me know.